dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -0,0 +1,314 @@
+from collections import defaultdict
+import itertools
+import re
+
+from . import utils, BaseColumnProfiler
+
+
+class TextProfiler(object):
+    type = 'unstructured_text'
+
+    def __init__(self, name, options=None):
+        """
+        Initialization of Text Profiler.
+
+        :param name: Name of the data
+        :type name: String
+        :param options: Options for the Text Profiler
+        :type options: UnstructuredTextOptions
+        """
+        self.name = name
+        self.sample_size = 0
+        self.times = defaultdict(float)
+        self.vocab = set()
+        self.word_count = defaultdict(int)
+        self.metadata = dict()
+
+        # TODO: Add line length
+        #self.line_length = {'max': None, 'min': None,...} #numeric stats mixin?
+
+        self._is_case_sensitive = True
+        if options:
+            self._is_case_sensitive = options.is_case_sensitive
+
+        # these stop words are from nltk
+        self._stop_words = {
```

Comment:
I created this myself from a merge of several different stop word datasets. If you see anything missing or things that shouldnt be in there, let me know, and I will change it.